app/src/main/java/org/jellyfin/androidtv/ui/AlphaPicker.kt

```diff
@@ -0,0 +1,41 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.Button
+import android.widget.LinearLayout
+import androidx.core.view.children
+import org.jellyfin.androidtv.R
+
+class AlphaPicker(
+	context: Context,
+	attrs: AttributeSet? = null
+) : LinearLayout(context, attrs) {
+	var onAlphaSelected: (letter: Char) -> Unit = {}
+
+	init {
+		removeAllViews()
+
+		val letters = "#${resources.getString(R.string.byletter_letters)}"
+		letters.forEach { letter ->
+			// NOTE: We must use a Button here and NOT AppCompatButton due to focus issues on Fire OS
+			val button = View.inflate(context, R.layout.alpha_picker_button, null) as Button
+			button.apply {
```

Comment:
```suggestion
			val binding = AlphaPickerButtonBinding.inflate(LayoutInflater.from(context))
			binding.button.apply {
```
(add id="button" or some other id to the XML)

Reply:
Actually use `val binding = AlphaPickerButtonBinding.inflate(LayoutInflater.from(context), this, false)` so it attaches to the parent to inherit styling (for themes)